src/api/provisionFormOptionsData.js

```diff
@@ -0,0 +1,33 @@
+module.exports = {
+	provisionFormOptions: 
+	[
+		{
+			services: [
+				{
+					identifier: 'github'
+				}
+			],
+			automationSoftwares: [
+				'ansible'
+			],
+			serverProviders: [
+				'digitalOcean'
```

Comment:
Yup. I agree @thetonymaster . IMHO snake_case would be the most appropriate convention for these identifiers...